src/event_handler.rs

```diff
@@ -105,8 +88,8 @@ where
     STATE: Clone,
 {
     pub app_state: &'a mut STATE,
-    pub transaction: &'a IncomingTransaction,
+    pub incoming_transaction: &'a IncomingTransaction,
+    pub incoming_event: &'a IncomingEvent,
     pub transaction_handle: &'a mut TRANSACTION_HANDLE,
```

Comment:
I guess we should have better names here.

Maybe better use `transaction_context` instead of `transaction_handle`

Reply:
Yep, this is something I was struggling with.

I wanted to name it transaction_context, but I didn't want it to be confused with the `TransactionHandlerContext`, which is the struct that is passed to transaction handlers.

Reply:
transaction handlers receive a TransactionHandlerContext
event handlers receive an EventHandlerContext

Now we also want to have a TransactionContext passed to event handlers, which would be a very logical name, but easy to confuse with the TransactionHandlerContext which is a very different thing.